web/style.css

```diff
@@ -120,12 +698,55 @@ h1, #msg-inner {
     background: white;
 }
 
+#load {
+    display: grid;
+    
+    width: 100%;
+    background-color: #ddd;
+
+    border-radius: 4px;
+    border: solid 1px black;
+}
+
+#load-bar {
+    grid-column: 1;
+    grid-row: 1;
+
+    width: 0%;
+    height: 30px;
+    background-color: #04AA6D;
+}
+
+#load-text {
+    grid-column: 1;
+    grid-row: 1;
+
+    text-align: center;
+    font-size: 15px;
+    line-height: 30px;
+    color: white;
```

Comment:
The color contrast is a bit bad here, maybe use `color:black`? You could also make the filled bar black, the empty bar white, and set the text color to `white` with `mix-blend-mode:difference`.

Reply:
The second option you provided gives a nice effect, I am just not sure whether a black loading bar is odd to the user. I will keep trying around a little.